portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -49,26 +50,21 @@ private String getComment(HttpServletRequest request) {
 
 @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String rn = request.getParameter("quantity");
-    int requestedNum = 5;
-
-    if (rn!=null) {
-        requestedNum = Integer.parseInt(rn);
-    }
+    String requestedNumString = request.getParameter("commentRequestedNum");
+    int requestedNumInt = Integer.parseInt(requestedNumString);
 
     Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
-    List<Entity> commentEntityList = results.asList(FetchOptions.Builder.withLimit(requestedNum));
-    List<Comment> commentList = new ArrayList<>();
+    List<Entity> commentEntityList = results.asList(FetchOptions.Builder.withLimit(requestedNumInt));
 
-    for (Entity entity : commentEntityList) {
-         long id = entity.getKey().getId();
-         String username = (String) entity.getProperty("username");
-         String commentText = (String) entity.getProperty("commentText");
-        commentList.add(new Comment(username, commentText, id));
-    }
+    List<Comment> commentList = commentEntityList
+                            .stream()
+                            .map(entity -> new Comment((String) entity.getProperty("username"),
+                                                   (String) entity.getProperty("commentText"),
+                                                   entity.getKey().getId()))
+                            .collect(toList());
```

Comment:
Out of curiosity, do you have a formatter set up? I imagine it would format this differently.

Reply:
Reformatted